### ISSUE TITLE
Add Redis bootstrap service key

### ISSUE THREAD

**Issue by andresrinivasan** [edgexfoundry/go-mod-core-contracts#285]
For Hanoi, there will be a Redis bootstrap service as described in https://github.com/edgexfoundry/edgex-go/issues/2503. In support of that service, a token will be issued to the service using a new key.

@andresrinivasan 
Hanoi

**Comment by jpwhitemn** [comment]
How does this work with / integrate with the security bootstrap ADR PR #140 on edgex-docs?

**Comment by andresrinivasan** [comment]
This simply adding a constant for the Redis bootstrap service.

```
diff --git a/clients/constants.go b/clients/constants.go
index fb311f6..0520025 100644
--- a/clients/constants.go
+++ b/clients/constants.go
@@ -69,6 +69,7 @@ const (
        SecuritySecretsSetupServiceKey      = "edgex-security-secrets-setup"
        SecurityProxySetupServiceKey        = "edgex-security-proxy-setup"
        SecurityFileTokenProviderServiceKey = "edgex-security-file-token-provider"
+       SecurityBootstrapRedisKey           = "edgex-security-bootstrap-redis"
 )
```

**Comment by jpwhitemn** [comment]
According to Andre - this is in support of https://github.com/edgexfoundry/edgex-go/issues/2503

**Comment by lenny-intel** [comment]
I am confused why this issue is submitted again this repo. Should it be on go-mod-bootstrap?

**Comment by andresrinivasan** [comment]
> I am confused why this issue is submitted against this repo. Should it be on go-mod-bootstrap?

Because I need a new service key and this is where the constants are being maintained.

**Comment by cloudxxx8** [comment]
@andresrinivasan is this fixed by #349?

**Comment by andresrinivasan** [comment]
Yes, this issue is resolved.
